include/SDL_cpuinfo.h

```diff
@@ -55,8 +55,15 @@
 #define __SSE2__
 #endif
 #endif /* __clang__ */
-#elif defined(__MINGW64_VERSION_MAJOR)
-#include <intrin.h>
+#elif defined(__MINGW32__) || defined(__MINGW64__)
```

Comment:
The `__MINGW64_VERSION_MAJOR` check is there to identify MinGW-w64
as opposed to MinGW.org.  Does MinGW.org provide intrin.h nowadays?
Otherwise, I don't see the need to change that check in to this.

Reply:
OK, I see where the original file would fail detecting mingw:
I suggest doing the following and leave the mingw-w64 detection
intact:
```diff
diff --git a/src/cpuinfo/SDL_cpuinfo.c b/src/cpuinfo/SDL_cpuinfo.c
index 1668e70..095bd0c 100644
--- a/src/cpuinfo/SDL_cpuinfo.c
+++ b/src/cpuinfo/SDL_cpuinfo.c
@@ -26,6 +26,9 @@
 
 #if defined(__WIN32__) || defined(__WINRT__)
 #include "../core/windows/SDL_windows.h"
+#ifdef __MINGW32__
+#include <_mingw.h>
+#endif
 #endif
 #if defined(__OS2__)
 #undef HAVE_SYSCTLBYNAME
```

Reply:
Isn't MinGW.org too old to use?

Reply:
> Isn't MinGW.org too old to use?

Maybe, but that's beside the point.

Reply:
Done as you said.